ckanext/ontario_theme/templates/internal/group/snippets/helper.html

```diff
@@ -0,0 +1,16 @@
+<div class="module module-narrow module-shallow">
+  <h2 class="module-heading">
+    <i class="fa fa-info-circle"></i>
+    {{ _('What are Collections?') }}
+  </h2>
+  <div class="module-content">
+    <p>
+      {% trans %}
+        You can create and manage collections of datasets.
+        This could be to catalogue datasets for a particular project or team,
+        or on a particular theme, or as a very simple way to help people find
+        and search your own published datasets.
+      {% endtrans %}
+    </p>
+  </div>
+</div>
```

Comment:
Could be it's own helper.html snippet for collections and load the right snippet based on collection type. May be overkill for this one but allows a bit more extensibility in the future. This may be more applicable to various dataset types if/when we get to those.